src/events.rs

```diff
@@ -87,6 +87,17 @@ pub enum WindowEvent {
 
     /// Touch event has been received
     Touch(Touch),
+
+    /// DPI scaling factor of the window has changed.
+    ///
+    /// The following actions cause DPI changes:
+    ///
+    /// * A user changes the resolution.
+    /// * A user changes the desktop scaling value (e.g. in Control Panel on Windows).
+    /// * A user moves the application window to a display with a different DPI. This message is sent when a window’s
+    /// position changes such that most of its area lies on a monitor with a DPI that is different from the DPI
+    /// before the position change.
```

Comment:
Actually this behavior will be platform dependent.

For example, on wayland, we don't have the information of how much the window covers a monitor. We just have the list of all monitors that display some part of the surface. Thus, the wayland backend will just return the highest dpi factor of all the monitors currently displaying some part of the window.

Reply:
I removed that part